client/web/src/auth/ExperimentalSignUpPage.tsx

```diff
@@ -22,12 +22,13 @@ interface Props extends ThemeProps, TelemetryProps {
 }
 
 const SourceToTitleMap = {
-    Context: 'Easily search the code you care about.',
+    Context: 'Easily search all the code you care about.',
     Saved: 'Create a library of useful searches.',
     Monitor: 'Monitor code for changes.',
     Extend: 'Augment code and workflows via extensions.',
-    SearchCTA: 'Add your public (and soon private) repositories.',
-    Snippet: 'Easily search the code you care about.',
+    SearchCTA: 'Add your public and private repositories.',
```

Comment:
```suggestion
    SearchCTA: 'Easily search the code you care about.',
```

I think this is the correct copy for this var as per this design:
https://www.figma.com/file/DzJsJq8DTBHUrauallnSwG/Public-code-CTA-updates?node-id=4%3A6168